README.md

```diff
@@ -56,8 +56,53 @@ The above workflow (when manually triggered) will
 ![image](https://user-images.githubusercontent.com/2510597/140506212-4938e44d-0662-4dc5-9fb1-c3f59fe075a6.png)
 
 - When you merge this PR, it will request an Npm OTP from Optic. (If you close the PR, nothing will happen)
-- Upon successful retrieval of the OTP, it will publish the package to Npm.
+- You need to define Npm and Optic tokens in the Github secrets for each user that will receive the otp
 - Create a Github release with change logs (You can customize release notes using [release.yml](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#example-configuration))
+- Upon successful retrieval of the OTP, it will publish the package to Npm.
+
+### Multiple user scenario
+
+In case there are multiple users who have access to trigger the release automation action, you can define npm and optic tokens for different
```

Comment:
let's be consistent with casing when mentioning npm and optic. let's use the same casing everywhere